src/components/ProgressForm/ProgressLayout.tsx

```diff
@@ -8,19 +8,31 @@ import ProgressForm from './ProgressForm';
 import styles from '@/components/ProgressForm/ProgressForm.module.scss';
 
 import { questions } from '@/constants/ProgressUpdates';
+import { getTotalMissedUpdates } from '@/utils/getTotalMissedUpdate';
+import { useUserProgressDetailsQuery } from '@/app/services/progressesApi';
+import { useGetUserQuery } from '@/app/services/userApi';
 
-function ProgressLayout() {
+const ProgressLayout: FC = () => {
+    const { data: user } = useGetUserQuery();
+    const { data: userStandupdata } = useUserProgressDetailsQuery(user?.id);
+    const standupDates = userStandupdata?.data?.map((element) => element.date);
+    const totalMissedUpdates = getTotalMissedUpdates(standupDates || []);
     return (
         <>
             <NavBar />
-            <ProgressHeader />
+            <div className={styles.progressHeader_container}>
+                <ProgressHeader
+                    totalMissedUpdates={totalMissedUpdates}
+                    updateType="Progress"
+                />
+            </div>
```

Comment:
I can see that the progress header internally is encapsulated in a div, how about adding the style class inside the progress header component's encapsulating div itself?

Reply:
I tried doing this and I think this cannot be done if this is to be done then it will require a lot of re-factoring and banner design of both the standup and progress update should be in sync then only we can do what you are saying and currently design of both are different.